### ISSUE TITLE
Donator Perks

### ISSUE THREAD

**Issue by NightHawk164** [drewhoener/JWSuggestions#57]
Not me! Again! :P 
By DeLawlz
1. For /s, being able to toggle silk touch or not to
2. A mobile furnace /f
3. Mobile Enchanted Table
4. Mobile Anvil

By rbnharper
1. /d for Elder Guardian
2. /v

**Comment by NightHawk164** [comment]
My personal remarks to these suggestions
1. For /s it'll actually be pretty cool to be able to toggles silk touch, but then what will be the point of silk touch picks? 
2. Mobile furnace sounds interesting, being able to move around knowing that you have something cooking in your furnace, infinitively 
3 and 4. I believe that these two items are a bit overpowered. We take about 31 iron ingots just to make a breakable anvil and for the Enchanted table, we need the bookshelves to give the enchanted table more levels as well as taking the diamond to make it.

1. /d Elder Guardian sounds good, but I'm just scared for the players that'll randomely see an elder guardian outside of an ocean temple
2. /v ........just no.

**Comment by johnnywoof** [comment]
All of the perks will not be added except for the elder guardian.

**Comment by DeLawlz** [comment]
What about a basic /chest?  Which you would be able to access at any time so you would have a bigger inventory pretty much.

**Comment by Spazgazum** [comment]
There's already /ec (ender chest).

**Comment by BlueKnightSam** [comment]
Yes all these perks are either too op (/v is way) or there. elder guardian is good though.

**Comment by 5d100** [comment]
mobile enchantment table is too op, as is stated before in a couple of other suggestions. 
Mobile furnace is a bit too over powered and a bit pointless as there's conflag and anything else can be done with /home.
I do support the others though.

**Comment by drewhoener** [comment]
@5d100 The ticket is closed but w/e xD.
A lot of these are actually just straight up impossible to do with the current resources given by the API (e.g. the anvil because it breaks). The others are just adding things because we can, which I'm not a supporter of. Let's give everyone a personal furnace because we can! There's no point, if you really need a furnace you can spend the time digging 8 blocks of cobble and getting some fuel
